tests/vmi_multus_test.go

```diff
@@ -219,6 +220,47 @@ var _ = Describe("Multus Networking", func() {
 			pingVirtualMachine(vmiOne, "10.1.1.2", "localhost:~#")
 		})
 	})
+
+	Context("Single VirtualMachineInstance with ovs-cni plugin interface", func() {
+		AfterEach(func() {
+			deleteVMIs(virtClient, []*v1.VirtualMachineInstance{vmiOne})
+		})
+
+		It("should report all interfaces in Status", func() {
+			interfaces := []v1.Interface{{Name: "default", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}},
+				{Name: "ovs", InterfaceBindingMethod: v1.InterfaceBindingMethod{Bridge: &v1.InterfaceBridge{}}}}
+			networks := []v1.Network{{Name: "default", NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}}},
+				{Name: "ovs", NetworkSource: v1.NetworkSource{Multus: &v1.CniNetwork{NetworkName: "ovs-net-vlan100"}}}}
+
+			vmiOne = createVMI(interfaces, networks)
+
+			tests.WaitUntilVMIReady(vmiOne, tests.LoggedInAlpineExpecter)
+
+			updatedVmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmiOne.ObjectMeta.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(len(updatedVmi.Status.Interfaces)).To(Equal(2))
+			interfacesByName := make(map[string]v1.VirtualMachineInstanceNetworkInterface)
+			for _, ifc := range updatedVmi.Status.Interfaces {
+				interfacesByName[ifc.Name] = ifc
+			}
+
+			for _, network := range networks {
+				ifc, is_present := interfacesByName[network.Name]
+				Expect(is_present).To(BeTrue())
+				Expect(ifc.MAC).To(Not(BeZero()))
```

Comment:
Let's add a check that the two mac addresses are not equal (easy to do and makes regressions a little bit less likely).